hack/verify-govet.sh

```diff
@@ -14,6 +14,10 @@ govet_blacklist=( "${OS_GOVET_BLACKLIST[@]-}" )
 
 function govet_blacklist_contains() {
 	local text=$1
+	if grep -Eqx "^(# github.com/openshift/origin.*|vet: typecheck failures)" <<<"${text}"; then
+		# there are certain lines that we must ignore and they are not exceptions
```

Comment:
Why do we ignore typecheck failures?

Reply:
That was when my blacklist was having items and that caused the blacklist not to work. I can remove them nowadays.